Riot/AppDelegate.m

```diff
@@ -1120,6 +1156,70 @@ - (void)application:(UIApplication *)application didRegisterUserNotificationSett
     }
 }
 
+// iOS 10+, see application:handleActionWithIdentifier:forLocalNotification:withResponseInfo:completionHandler:
+- (void)userNotificationCenter:(UNUserNotificationCenter *)center didReceiveNotificationResponse:(UNNotificationResponse *)response withCompletionHandler:(void (^)(void))completionHandler
+{
+    UNNotification *notification = response.notification;
+    UNNotificationContent *content = notification.request.content;
+    if ([[response actionIdentifier] isEqualToString:@"inline-reply"]) {
+        UNTextInputNotificationResponse *textResponse = (UNTextInputNotificationResponse *) response;
```

Comment:
Note:
Could be interesting to factorize inline reply code between the two APIs with a method signature like this for example:
`- (void)handleNotificationInlineReplyForRoomId:(NSString*)roomId withResponseText:(NSString*)responseText success:(void(^)(NSString *eventId))success failure:(void(^)(NSError *error))failure`.